app/api/orders.py

```diff
@@ -40,6 +40,15 @@
 alipay_blueprint = Blueprint('alipay_blueprint', __name__, url_prefix='/v1/alipay')
 
 
+def check_ticket_holders(order, data, element):
+    ticket_holders = []
```

Comment:
Where is this used?

Reply:
This is being used on line 46 to append ticket_holder.id

Reply:
And then where is it used? After appending the IDs?

Reply:
On line 52 to check if `data['ticket_holders'] == ticket_holders`.